### PR TITLE
Publish Docker image

### PR DESCRIPTION
**Pull request by giorgiosironi**
For https://github.com/libero/publisher/issues/310

The script should be later extracted into the `scripts` submodule.

### CODE REVIEW

**Comment by giorgiosironi** [comment]
Added `DOCKER_*` secrets at https://github.com/libero/article-store/settings/secrets

File: .github/push-image.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```
Comment (giorgiosironi) [inline .github/push-image.sh]: compare https://github.com/libero/scripts/blob/master/travis/push-image.sh

Comment (thewilkybarkid) [reply .github/push-image.sh]: We should move this there (`libero/script`).

If it needs testing, let's do it on a dummy repo.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -80,3 +80,27 @@ jobs:
 
       - name: Run test
         run: ${{ matrix.test }}
+
+  push:
+
+    needs: test
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
```
Comment (thewilkybarkid) [inline .github/workflows/ci.yml:89]: Shame there's no simpler way.

Comment (giorgiosironi) [reply .github/workflows/ci.yml:89]: The event types would limit to pushes, but not to a particular branch.

File: .github/push-image.sh (hunk 1 above)
Comment (giorgiosironi) [reply .github/push-image.sh]: It should ideally tested in the `script` repository itself, to validate PRs.

Comment (giorgiosironi) [reply .github/push-image.sh]: Working on https://github.com/libero/scripts/pull/9 to eventually do that

**Comment by giorgiosironi** [comment]
Compare https://github.com/libero/article-store/settings/secrets. Can't keep it up-to-date with `master` enough, but as ready as it can be.

**Review by thewilkybarkid — APPROVED** [review]
Looks like it should work.

**Pull request merged**